test/WebSites/SimpleWebSite/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "commands": {
+    "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
+    "kestrel": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.Kestrel --server.urls http://localhost:5000"
```

Comment:
:ok: but FYI all but "BasicWebSite" that I looked at uses the other format. I guess that's old and busted and yours is the new hotness?